Makefile

```diff
@@ -384,7 +384,7 @@ clean-launch-loki:
 	helm uninstall loki
 
 launch-dex:
-	helm repo add --force-update stable https://charts.helm.sh/stable
```

Comment:
I'm not sure what the difference is between the version of helm in testfaster and the version in Travis - I think we're OK to drop this for now. Once the identity server alpha lands these will use the identity server instead of dex anyways.